src/lib/mongoBackend/mongoSubscribeContext.cpp

```diff
@@ -183,10 +183,12 @@ HttpStatusCode mongoSubscribeContext
     {
       std::string oidString = oid.toString();
 
-      orion::subCache->insert(new orion::Subscription(tenant, servicePath, requestP, oidString, expiration, notifyFormat));
+      LM_T(LmtMongoSubCache, ("inserting a new sub in cache (%s)", oidString.c_str()));
+      mongoSubCacheItemInsert(tenant.c_str(), servicePath.c_str(), requestP, oidString.c_str(), expiration, throttling, notifyFormat);
     }
 
 
+    reqSemGive(__FUNCTION__, "ngsi10 subscribe request", reqSemTaken);
```

Comment:
This one is probably me - look at the call to mongoSubCache.
Those where mongoSubCache is found, those moves of the semaphore are mine.
If no mongoSubCache is there ... then it's not mine